package.json

```diff
@@ -113,6 +112,7 @@
     "terra-infinite-list": "^3.0.0",
     "terra-legacy-theme": "^2.0.0",
     "terra-list": "^4.0.0",
+    "terra-markdown": "^2.0.0",
```

Comment:
this can be removed now too.

Reply:
Removed https://github.com/cerner/terra-ui/pull/310/commits/e9da608d29aaca7a6a54bf05797065b88e72b842#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L116